src/test/performance/logistic_test.cpp

```diff
@@ -0,0 +1,234 @@
+#include <gtest/gtest.h>
+#include <test/test-models/performance/logistic.hpp>
+#include <boost/date_time/posix_time/posix_time.hpp>
+#include <test/performance/utility.hpp>
+#include <boost/algorithm/string/trim.hpp>
+
+
+class performance : public ::testing::Test {
+public:
+  static void SetUpTestCase() {
+    N = 100;
+    milliseconds_per_run.resize(N);
+    last_draws_per_run.resize(N);
+    matches_tagged_version = false;
+    all_values_same = false;
+  }
+
+  static int N;
+  static Eigen::Matrix<long, -1, 1> milliseconds_per_run;
```

Comment:
Why Eigen types here rather than std::vector?  I also prefer `Eigen::Dynamic` to `-1`, but it's no big deal.